inc/admin/class-cenote-theme-review-notice.php

```diff
@@ -4,14 +4,14 @@
  *
  * @author  ThemeGrill
  * @package Cenote
- * @since   1.1.9
+ * @since   1.3.2
```

Comment:
No need to change the @since as there previously.

Reply:
Actually the since version was from the other theme from where I took reference.

Reply:
Until now there is no Cenote_Theme_Review_Notice class